src/org/dotnet/ci/triggers/GenericTriggerBuilder.groovy

```diff
@@ -48,19 +58,23 @@ class GenericTriggerBuilder implements TriggerBuilder {
     // Parameters:
     //  job - Job to emit trigger for
     void emitTrigger(def job) {
-        assert this._triggerType == TriggerType.PERIODIC
+        assert (this._triggerType == TriggerType.PERIODIC) || (this._triggerType == TriggerType.MANUAL)
```

Comment:
You could consider refactoring this into something like

```
if (this._triggerType == TriggerType.Periodic) {
 ...
}
else if (this._trigerType == TriggerType.Manual) {
 ...
}
else {
    assert(blah) /* Unsupported trigger type */
}
```

May simplify adding trigger types in the future.